src/components/mainContent/ForwardButton.js

```diff
@@ -44,8 +45,12 @@ export default function ForwardButton(props) {
     } 
     else 
     {
+      if (props.currentVisibleText === 0) {
+        const section = props.content[props.visibleSections];
+        scroll.scrollTo((section.post ? section.post.header : section.followups[0]), {spy: true, smooth:true, offset:-100, duration: 500});
```

Comment:
Just a quick note here: do you mind documenting what some of these configuration options do? Since this is an external component, it can make it easier for future developers to change it!

(since these constants are duped, we may want to abstract it out)